defaults/main.yml

```
@@ -51,6 +51,7 @@ lephare_run_doctrine_migrations: true
 lephare_console_path: bin/console
 lephare_composer_options: "--no-dev --optimize-autoloader --apcu-autoloader --no-interaction"
 lephare_composer_dump_environment: true
+lephare_composer_home: '~/.composer'
```

Depuis Composer 2 ça s'installe dans `${XDG_CONFIG_HOME-.config}/composer`

```suggestion
lephare_composer_home: '~/.config/composer'
```

---

Oui il s'agit de la valeur par défaut qui serait surchargée par $COMPOSER_HOME https://getcomposer.org/doc/03-cli.md#composer-home

Peu m'importe